dolphinscheduler-ui/src/views/projects/workflow/components/dag/index.tsx

```diff
@@ -233,23 +237,28 @@ export default defineComponent({
     const handleViewLog = (taskId: number, taskType: string) => {
       taskModalVisible.value = false
       viewLog(taskId, taskType)
-      getLogs()
+      getLogs(logTimer)
     }
 
-    const getLogs = () => {
+    const getLogs = (logTimer: number) => {
       const { state } = useAsyncState(
         queryLog({
           taskInstanceId: nodeVariables.logTaskId,
           limit: nodeVariables.limit,
           skipLineNum: nodeVariables.skipLineNum
         }).then((res: any) => {
-          if (res.message) {
-            nodeVariables.logRef += res.message
+          nodeVariables.logRef += res.message || ''
+          if (res && res.message !== '') {
             nodeVariables.limit += 1000
             nodeVariables.skipLineNum += res.lineNum
-            getLogs()
+            getLogs(logTimer)
           } else {
             nodeVariables.logLoadingRef = false
+            setTimeout(() => {
+              nodeVariables.limit += 1000
+              nodeVariables.skipLineNum += 1000
+              getLogs(logTimer)
+            }, logTimer * 1000)
```

Comment:
Same as above.

Reply:
Addressed in https://github.com/apache/dolphinscheduler/pull/11226/commits/fcfe28f870b4fcb367fa51591e0136fe040b382a.